StatPlugin.cs

```diff
@@ -1,40 +1,37 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
+using ServiceStack.Redis;
 
 namespace DNWS
 {
   class StatPlugin : IPlugin
   {
+    protected static RedisManagerPool manager = null;
     protected static Dictionary<String, int> statDictionary = null;
     public StatPlugin()
     {
-      if (statDictionary == null)
-      {
-        statDictionary = new Dictionary<String, int>();
-
-      }
+      manager =  new RedisManagerPool("redis:6379");
```

Comment:
it's weird that manager is a static, but you instantiate it every time you instantiate StatePlugin class.